test/K6/src/data/testdata/access-management/rights-delegations/localtestdata.json

```diff
@@ -15,6 +15,11 @@
             "pid": "17048001742",
             "userid": "20003595",
             "partyid": "50020440"
+        },
+        "apiadm": {
```

Comment:
API adm alone won't give access to the Rights DelegationCheck. Separate endpoint is being made for MaskinportenSchema DelegationCheck

Reply:
Removed these testdata